src/main/java/org/jmhsrobotics/frc2024/subsystems/intake/commands/IntakeCommand.java

```diff
@@ -40,7 +40,7 @@ public void initialize() {
 	@Override
 	public void execute() {
 		this.intakeSubsystem.set(this.speed);
-		this.shooterSubsystem.set(-.05, ControlType.VOLTAGE);
+		this.shooterSubsystem.set(-0.02, ControlType.VOLTAGE);
```

Comment:
Why do we have this line here. 0.02 volts is not going to be enough the move the motor. the range should be +-13 volts ish. So 10% power would be like 1.3 volts.